src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2370,6 +2372,7 @@ def create_policy_definition(cmd, name, rules=None, params=None, display_name=No
         enforce_mutually_exclusive(subscription, management_group)
         if management_group:
             policy_client = _resource_policy_client_factory(cmd.cli_ctx)
+            print(parameters)
```

Comment:
May I ask why do we need to print the parameters?

Reply:
deleted